app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.jsx

```diff
@@ -158,6 +160,7 @@ export class InputConditionalAttributes extends Component {
             projectId={projectId}
             onConfirm={this.onChangeAttributes}
             nakedView
+            attribute={{ edited }}
```

Comment:
I am not sure, that it is a good solution.
May it would be better to provide a default value in defaultProps?
It would be nice, to describe the whole structure of attribute in propTypes.
And after that provide default values for that prop.

Reply:
fixed ✓